pghoard/rohmu/object_storage/s3.py

```diff
@@ -147,19 +152,21 @@ def store_file_from_disk(self, key, filepath, metadata=None, multipart=None):
         size = os.path.getsize(filepath)
         key = self.format_key_for_backend(key)
         metadata = self.sanitize_metadata(metadata)
-        if not multipart or size <= self.multipart_chunk_size:
+        if not multipart and size <= self.multipart_chunk_size:
```

Comment:
Could you explain why you want to change this from an or to an and again? I gave this a try and the tests pass with both variants but I'm wondering why you'd want to change this bit.